.github/workflows/backend.yaml

```diff
@@ -4,7 +4,15 @@ name: Backend CI
 on:
   push:
     branches: [main]
+    paths-ignore:
```

Comment:
As already discussed, let's use the positive `path` matching.

Reply:
just remove my last commit, and it's now erady